src/utils/dateFormatter.ts

```diff
@@ -0,0 +1,7 @@
+import { format } from 'date-fns';
+
+const DATE_FORMAT = 'EEE d MMMM y';
```

Comment:
(Suggestion) And changing the date format:
```suggestion
const DATE_FORMAT = 'dd/MM/yyyy';
```